### PR TITLE
Fixed #27032 -- Added error if setup_test_environment() called twice.

### PR DESCRIPTION
**Pull request by cjerdonek**

### CODE REVIEW

File: docs/releases/1.11.txt [hunk 1]
```diff
@@ -364,6 +364,10 @@ Miscellaneous
   :meth:`~django.db.models.fields.related.RelatedManager.set` now
   clear the ``prefetch_related()`` cache.
 
+* :func:`~django.test.utils.setup_test_environment` now raises an exception
+  if called a second time before calling
+  :func:`~django.test.utils.teardown_test_environment`.
+
```
Comment (claudep) [inline docs/releases/1.11.txt]: Is this really worth a release note?

Comment (cjerdonek) [inline docs/releases/1.11.txt]: @claudep, @timgraham requested it here: https://github.com/django/django/pull/7032#issuecomment-237967038

Comment (claudep) [inline docs/releases/1.11.txt]: OK, fine.

**Comment by timgraham** [comment]
merged in e7fb724cd2928802fbefa3b5491640fab51e898a, thanks!

**Pull request closed without merging**